templates/clusterclass/clusterclass.yaml

```diff
@@ -146,10 +141,10 @@ spec:
             port: {{ .prismCentralEndpoint.port }}
             insecure: {{ .prismCentralEndpoint.insecure }}
             credentialRef:
-              name: "${CLUSTER_NAME}"
+              name: '{{ .prismCentralEndpoint.credentialSecret }}'
```

Comment:
Why are these go templates?

Reply:
I don't think they need to be here.

Reply:
as discussed with @thunderboltsid this is as expected